main.js

```diff
@@ -0,0 +1,203 @@
+/*
+  Project Shift Week 1 Project - Tic-Tac-Toe
+  Austin Stevens
+*/
+
+/*
+  Players array to track both players
+  Each player has a name, character of X or O, and a status to indicate
+  whether he/she is able to play
+*/
+var TicTacToe = function() {
+  var board = [
+    ["1", "2", "3"],
+    ["4", "5", "6"],
+    ["7", "8", "9"]
+  ];
+  var currentPlayer = player1;
+  var gridSize = 3;
+  var moveCount = 0; // Tracking the number of turns taken.
+  var nameSet = false; // Indicates that setup is completed with valid names.
```

Comment:
I'm not sure this is the best name for this variable as it's being used for several things.